rts/lib/sol2/sol.hpp

```diff
@@ -9146,7 +9146,7 @@ namespace sol {
 
 	inline int push_type_panic_string(lua_State* L, int index, type expected, type actual, string_view message, string_view aux_message) noexcept {
 		const char* err = message.size() == 0
-		     ? (aux_message.size() == 0 ? "stack index %d, expected %s, received %s" : "stack index %d, expected %s, received %s: %s")
+		     ? (aux_message.size() == 0 ? "stack index %d, expected %s, received %s" : "stack index %d, expected %s, received %s: %s%s")
```

Comment:
Just want to check: is this formatting correct? two '%s%s' with no space at the end? I.e there are two strings combined that do not require space between them?

Reply:
this is correct, it's the same change that was upstreamed several months ago

https://github.com/ThePhD/sol2/commit/e8e122e9ce46f4f1c0b04003d8b703fe1b89755a